### PR TITLE
issue #37: Raising an error when adding al already registered package

### PR DESCRIPTION
**Pull request by ppoilbarbe**
Here is the merge request (my first one so I am not sure I do it right) as proposed in issue #37

### CODE REVIEW

File: man/man1/freight-add.1 [hunk 1]
```diff
@@ -26,6 +26,10 @@ Use an alternate configuration file\.
 Verbose mode\.
 .
 .TP
+\fB\-e\fR, \fB\-\-add\-error\fR
```
Comment (ppoilbarbe) [inline man/man1/freight-add.1]: No. I did not see in the makefile that the files are generated. So I found a bit obscure modifying directly the man files.
The .ronn file is modified and pushed again.

**Comment by mattock** [comment]
Why we are versioning the man files if they are dynamically generated from the .ronn files? Doing the conversion on-the-fly as a packaging step would probably make more sense. Thoughts?

As for this PR: it would be nice if the two commits would be squashed into one, so that Git history would remain cleaner. Of the top of my hat it goes something like this - do this on your local copy:

    $ git reset --soft <commit-id-before-these-patches>

Then make edits and reverts as necessary (git status will give good instructions):

    $ git commit
    $ git push -f origin/master

GitHub will then automagically notice that the PR has been updated.

**Comment by ppoilbarbe** [comment]
I understand that having the man files under git and not only the ronn file avoids that everybody which try to install freight does not have to install the ronn command (and its dependencies), making freight standalone.
I am not very comfortable with these git manipulations, and I am not sure I did suppress previous commits: it seems that there is 2 new ones...

**Comment by mattock** [comment]
I can create a cleaned up PR and credit you as  the author, if @domcleal and @mmoll are fine with the general contents of this PR.

**Comment by domcleal** [comment]
Fine by me if you're like to @mattock, or just go ahead and commit the squashed version. It might also be possible to use GitHub squash merge feature, but I'm unsure if it deals with merge commits in the PR branch.

**Comment by ppoilbarbe** [comment]
Thanks... I'll learn how to things right with git (at now I only use basic commands and git flow).

**Comment by Daniel15** [comment]
> it would be nice if the two commits would be squashed into one

Github can do that for you when you merge the pull request. No need to get contributors to manually squash their commits 😄

**Comment by mmoll** [comment]
merged, thanks @ppoilbarbe 

unfortunately the GitHub feature did not work as intended in this one, so the next time I'll squash this manually before merging again...

**Pull request merged**